fastlane-plugin-github_job_status/spec/github_job_status_action_spec.rb

```diff
@@ -25,6 +25,7 @@
         repo: 'fastlane_plugins',
         sha: 'commit_sha',
         job_name: 'good_job',
+        description: 'A short description',
```

Comment:
Did you try running this test? It looks like this should fail since the description in the payload is 'Job pending'.

Also, please make providing a description a separate test case so that we still have test coverage of the default description.